app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,11 @@
+class FoodsController < ApplicationController
+  def search
+    @total_results = FoodDetailsService.new.keyword_search(params[:q])[:totalHits]
+    foods = FoodDetailsService.new.keyword_search(params[:q])[:foods][0..9]
```

Comment:
You're calling to the same service method twice, and filtering out information to be stored in variables. Filtering is the job of a Facade. Also, if you're calling to the same endpoint twice, you should find a way to call to it once, and then pick out total results and the foods.